src/clockcache.c

```diff
@@ -915,12 +920,20 @@ clockcache_assert_no_locks_held(clockcache *cc)
 void
 clockcache_assert_clean(clockcache *cc)
 {
-   uint64 i;
+#if SPLINTER_DEBUG
 
-   for (i = 0; i < cc->cfg->page_capacity; i++) {
-      debug_assert(clockcache_test_flag(cc, i, CC_FREE)
-                   || clockcache_test_flag(cc, i, CC_CLEAN));
+   for (uint64 i = 0; i < cc->cfg->page_capacity; i++) {
+
+      // We expect entry to be in only one of these two states.
+      entry_status entry_flag = clockcache_get_flag(cc, i);
+      debug_assert(((entry_flag & (CC_FREE | CC_CLEAN)) != 0),
+                   "Buffer at entry=%lu should be in either CC_FREE|CC_CLEAN"
+                   " status. Found unexpected status=0x%x %s\n",
+                   i,
+                   entry_flag,
+                   ((entry_flag & CC_ACCESSED) ? "(CC_ACCESSED)" : ""));
```

Comment:
Expanded debug asserts ... Was running into this before I figured out the need for the fix to `mmap()`. 

Leave it.... will benefit some future engineer.